README.md

```diff
@@ -1 +1,129 @@
-# WashingBot
+Welcome to WashingBot!
+----------------------
+
+A [DroidScript](https://play.google.com/store/apps/details?id=com.smartphoneremote.androidscriptfree) app to help you with your laundry. WashingBot monitors your washing machine, keeps you updated, reminds you to hang your clothes, talks and you can even chat with him!
+
+Low water pressure, different washing options and your mental state can affect the duration of your washing cycle, if you like us are too lazy to care and hate going to check in vane, then this app is for you!
+
+![WashingBot!](https://github.com/MetalBunker/WashingBot/blob/master/Img/WashingBot.png)
+  
+WashingBot feels the vibrations of your washing machine by using your cell's accelerometer, so he knows (or tries to guess :P) how is your washing going. He will notify you when your washing starts, finishes and he will even remind you to go and hang your clothes!  
+Notifications are verbal (yes, he likes to talk) and by chat, using [Slack](https://slack.com/). You can even ask him for how long he has been washing (yeah, he thinks he's actually doing the laundry, be nice and don't tell him).
```

Comment:
After finishing reading, I realized that you haven't used emoticons everywhere. No need to change this, but if you do change it, please look for all the other places.